flink-runtime/src/main/java/org/apache/flink/runtime/io/network/netty/CreditBasedSequenceNumberingViewReader.java

```diff
@@ -206,7 +201,7 @@ public String toString() {
 		return "CreditBasedSequenceNumberingViewReader{" +
 			"requestLock=" + requestLock +
 			", receiverId=" + receiverId +
-			", buffersAvailable=" + buffersAvailable.get() +
+			", buffersAvailable=" + hasBuffersAvailable() +
```

Comment:
this line should now be removed (the property is not part of this object anymore and going through a synchronized block which may be nasty during debugging sessions)